Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:alpine
+
+RUN apk --update add --virtual build_deps \
+    build-base \
+    libc-dev \
+    linux-headers \
+    cmake \
+    && apk --no-cache add icu-dev libressl-dev \
+    && gem install github-linguist \
+    && apk del build_deps
```

Comment:
Any reason to not uninstall the other packages? It looks like we don't need them anymore and it would make the image smaller.

Reply:
Good point, removed useless

Reply:
I think you need those packages to build Linguist (at least `build-essential`). I was just suggesting we remove them after the installation.

Reply:
Fixed